read-fonts/src/collections/int_set/mod.rs

```diff
@@ -968,6 +969,33 @@ impl Domain for GlyphId {
     }
 }
 
+impl Domain for Tag {
+    fn to_u32(&self) -> u32 {
+        u32::from_be_bytes(self.to_be_bytes())
+    }
+
+    fn from_u32(member: InDomain) -> Tag {
+        Tag::from_u32(member.value())
+    }
+
+    fn is_continuous() -> bool {
+        true
+    }
+
+    // Tag is 4 bytes, each in the ascii range (`0x20..=0x7E`)
+    fn ordered_values() -> impl DoubleEndedIterator<Item = u32> {
+        0x20202020..=0x7E7E7E7E
```

Comment:
It looks like it's possible to construct Tag's with values outside of ASCII so ordered_values() and count should just use u32::MIN and u32::MAX.